js/Update.elm

```diff
@@ -116,12 +110,16 @@ updateProduction msg m =
             )
 
 
-updateAuction : AuctionMsg -> AuctionModel -> ( AuctionModel, Cmd Msg )
-updateAuction msg m =
+type alias SendToServer =
```

Comment:
we can actually compose `send` with `encodeToMessage` to abstract away the encoding, and let `send` directly take in an `Api.Action`